skimage/restoration/tests/test_denoise.py

```diff
@@ -172,6 +172,16 @@ def test_denoise_bilateral_2d():
     assert_(out1[30:45, 5:15].std() > out2[30:45, 5:15].std())
 
 
+def test_denoise_bilateral_zeros():
+    img = np.zeros((10, 10))
+    assert_equal(img, restoration.denoise_bilateral(img))
+
+
+def test_denoise_bilateral_constant():
+    img = np.ones((10, 10)) * 5
+    assert_equal(img, restoration.denoise_bilateral(img))
```

Comment:
Oops, I missed this the first time: it looks like this call needs a `multichannel=False` kwarg, or the test fails with a `ValueError`.